BiometricLoginKotlin/app/src/main/java/com/example/biometricloginsample/CryptographyManager.kt

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+
+package com.example.biometricloginsample
+
+import android.content.Context
+import android.security.keystore.KeyGenParameterSpec
+import android.security.keystore.KeyProperties
+import com.google.gson.Gson
+import java.nio.charset.Charset
+import java.security.KeyStore
+import javax.crypto.Cipher
+import javax.crypto.KeyGenerator
+import javax.crypto.SecretKey
+import javax.crypto.spec.GCMParameterSpec
+
+interface CryptographyManager {
+
+    /**
+     * This method first gets or generates an instance of SecretKey and then initializes the Cipher
+     * with the key. The secret key uses [ENCRYPT_MODE][Cipher.ENCRYPT_MODE] is used.
+     */
+    fun getInitializedCipherForEncryption(keyName: String): Cipher
+
+    /**
+     * This method first gets or generates an instance of SecretKey and then initializes the Cipher
+     * with the key. The secret key uses [DECRYPT_MODE][Cipher.DECRYPT_MODE] is used.
```

Comment:
`The secret key uses [DECRYPT_MODE][Cipher.DECRYPT_MODE] is used.` Does it have to be used?